docs/Modding/Tilesets/graphics.rst

```diff
@@ -0,0 +1,446 @@
+Tile Specification Files
+************************
+
+Creating a Freeciv21 Tileset requires a mix of editing plain text :file:`spec` files as well as creating
+graphics files saved in the :file:`.png` image format. When complete, a Tileset editor will have created
+the canvas that Freeciv21 players play upon. Next to :doc:`../Rulesets/index`, creating Tilesets is one of the
+most important aspects of a Freeciv21 game.
+
+Using Graphics
+---------------
+
+To use different graphics with Freeciv21, use the :code:`--tiles` argument to the Freeciv21 client. Eg, to use
+the 'engels' graphics, start the client as:
+
+.. code-block:: rst
+
+    freeciv21-client --tiles engels
+
+
+What Freeciv21 actually does in this case is look for a file called :file:`engels.tilespec` somewhere in your
+Freeciv21 data path. That :file:`.tilespec` file contains information telling Freeciv21 which graphics files
+to use, and what those graphics files contain.
+
+That is all you need to know to use alternative graphics provided by Freeciv21 or by third-party add-ons. The
```

Comment:
Better advertise the GUI settings if not for tileset authors. And for third-party addons, a link to the Modpack installer could be provided.